proto/decentraland/sdk/components/ui_background.proto

```diff
@@ -5,7 +5,30 @@ import "decentraland/sdk/components/common/id.proto";
 option (common.ecs_component_id) = 1053;
 
 import "decentraland/common/colors.proto";
+import "decentraland/common/texture.proto";
+import "decentraland/common/border_rect.proto";
 
 message PBUiBackground {
-  optional decentraland.common.Color4 background_color = 1; // default=(0.0, 0.0, 0.0, 0.0)
-}
\ No newline at end of file
+  optional decentraland.common.Color4 color = 1; // default=(1.0, 1.0, 1.0, 1.0), pixel = color * sample2D(texture, uv)
+  optional decentraland.common.Texture texture = 2;
+  BackgroundTextureMode texture_mode = 3;
```

Comment:
```suggestion
  BackgroundTextureMode background_texture_mode = 3;
```

can skip this if you considered it's already ok!

Reply:
we will not have any colliding names in the same context